fideslib/oauth/scopes.py

```diff
@@ -31,13 +37,22 @@
 CONNECTION_CREATE_OR_UPDATE = f"{CONNECTION}:{CREATE_OR_UPDATE}"
 CONNECTION_DELETE = f"{CONNECTION}:{DELETE}"
 CONNECTION_READ = f"{CONNECTION}:{READ}"
+CONNECTION_AUTHORIZE = "connection:authorize"
```

Comment:
Just for consistency. 

```suggestion
AUTHORIZE = "authorize"

CONNECTION_AUTHORIZE = f"{CONNECTION}:{AUTHORIZE}"
```